src/MainWindow.vala

```diff
@@ -120,4 +126,48 @@ public class Dock.MainWindow : Gtk.ApplicationWindow {
             return false;
         });
     }
+
+    public void move_launcher_after (Launcher source, Launcher? target) {
```

Comment:
I can't understand what it's trying to do. Maybe add some comments?

Reply:
I added a comment not sure whether it's understandably written though  😅

Reply:
Oh, sorry I means the code inside the method, I genuinely don't understand what variable `add` for and why it used in weird `add = !add;`.

Reply:
Oh right I added some comments and renamed the variable (no idea why I named it `add` in the first place :))